### PR TITLE
NET7

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -16,10 +16,10 @@ jobs:
     - name: Checkout source
       uses: actions/checkout@v3
 
-    - name: Setup .NET Core SDK
-      uses: actions/setup-dotnet@v3
+    - name: Setup .NET SDK
+      uses: actions/setup-dotnet@v3.0.3
```
Comment (Shane32) [inline .github/workflows/codeql-analysis.yml:20]: Why pinning to 3.0.3?

Comment (sungam3r) [reply .github/workflows/codeql-analysis.yml:20]: Why not?  Just updated as here https://github.com/graphql-dotnet/graphql-dotnet/blob/master/.github/workflows/codeql-analysis.yml#L20

Comment (Shane32) [reply .github/workflows/codeql-analysis.yml:20]: Because then you won’t get v3.0.4

Comment (sungam3r) [reply .github/workflows/codeql-analysis.yml:20]: I don't understand. Dependabot bumps this version from time to time: 2 -> 3 -> 3.0.1 -> 3.0.2 -> 3.0.3

Comment (Shane32) [reply .github/workflows/codeql-analysis.yml:20]: That’s just the tag. “v3” is updated when there’s a new release. See here how the sha is the same for v3 and v3.0.3

https://github.com/actions/setup-dotnet/tags

Comment (Shane32) [reply .github/workflows/codeql-analysis.yml:20]: Ignore dependabot

Comment (sungam3r) [reply .github/workflows/codeql-analysis.yml:20]: It means I don't know how versioning works in actions but I just did the same as we did in other repos.

Comment (Shane32) [reply .github/workflows/codeql-analysis.yml:20]:
```suggestion
      uses: actions/setup-dotnet@v3
```

File: .github/workflows/publish-preview.yml [hunk 2]
```diff
@@ -21,10 +21,10 @@ jobs:
     steps:
       - name: Checkout source
         uses: actions/checkout@v3
-      - name: Setup .NET Core 6.0 SDK
-        uses: actions/setup-dotnet@v3
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v3.0.3
```
Comment (Shane32) [inline .github/workflows/publish-preview.yml:25]:
```suggestion
        uses: actions/setup-dotnet@v3
```

File: .github/workflows/publish-release.yml [hunk 3]
```diff
@@ -27,10 +27,10 @@ jobs:
           version="${github_ref:10}"
           echo version=$version
           echo "version=$version" >> $GITHUB_ENV
-      - name: Setup .NET Core 6.0 SDK
-        uses: actions/setup-dotnet@v3
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v3.0.3
```
Comment (Shane32) [inline .github/workflows/publish-release.yml:31]:
```suggestion
        uses: actions/setup-dotnet@v3
```

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -31,13 +31,14 @@ jobs:
     steps:
       - name: Checkout source
         uses: actions/checkout@v3
-      - name: Setup .NET Core SDKs
-        uses: actions/setup-dotnet@v3
+      - name: Setup .NET SDKs
+        uses: actions/setup-dotnet@v3.0.3
```
Comment (Shane32) [inline .github/workflows/test.yml:35]:
```suggestion
        uses: actions/setup-dotnet@v3
```

**Review by Shane32 — APPROVED** [review]

**Pull request merged**